core/roboconf-agent/src/main/java/net/roboconf/agent/internal/Agent.java

```diff
@@ -82,7 +81,7 @@
 	 */
 	public Agent() {
 		this.logger = Logger.getLogger( getClass().getName());
-		this.messagingFactoryType = MessagingConstants.FACTORY_RABBIT_MQ;
+		this.messagingType = MessagingConstants.FACTORY_RABBIT_MQ;
```

Comment:
Still working on it. Now the tough part seams to be fixed, I'm dealing with refactoring and other "light" fixes.